src/Listener/JsonApiListener.php

```diff
@@ -341,6 +344,65 @@ public function beforeFind(Event $event)
         }
     }
 
+    /**
+     * Add 'sort' capability
+     *
+     * @see http://jsonapi.org/format/#fetching-sorting
+     * @param string|array $sortFields Field sort request
+     * @param \Crud\Event\Subject $subject The subject
+     * @param array $options Array of options for includes.
+     * @return void
+     */
+    protected function _sortParameter($sortFields, Subject $subject, $options)
+    {
+        if (is_string($sortFields)) {
+            $sortFields = explode(',', $sortFields);
+        }
+        $sortFields = array_filter((array)$sortFields);
+
+        $order = [];
+        $includes = $this->config('include');
```

Comment:
Can you explain the `include` functionality?

Reply:
As far as I know, this is an existing functionality, processed by `_includeParamer` in  ff033cb. You basically pass an comma separated values of valid types that is related to the type in context, eg:

`/countries?include=currencies,national_cities`

The resulting response should include the currencies + national cities in the payload.

From reading the code,  those values are read from the request and set accordingly in https://github.com/FriendsOfCake/crud-json-api/blob/1fe01b0849d4074ddf3c6a38e7cb20890d24c1d5/src/Listener/JsonApiListener.php#L311

Reply:
That makes sense, I think that would be part of the functionality added by mr. @dakota.